selene_sdk/utils/config_utils.py

```diff
@@ -205,79 +207,48 @@ def create_data_source(configs, output_dir=None, load_train_val=True, load_test=
         dataset_info["dataset_args"]["target_features"] = target_features
         dataset_info["dataset_args"]["distinct_features"] = distinct_features
 
-        if load_train_val:
-            # load train dataset and loader
-            train_config = dataset_info["dataset_args"].copy()
-            train_config["intervals"] = train_intervals
-            if "train_transform" in dataset_info:
+        # create datasets, samplers, and loaders
+        tasks = ["train","validation","test"]
```

Comment:
```suggestion
        tasks = []
        if load_train_val:
            tasks.extend(["train", "val"])
        if load_test:
            tasks.append("test")
```

Reply:
changed 
`tasks.extend(["train", "val"])`
to
`tasks.extend(["train", "validation"])`